_pages/algorithm-development/your-first-algo.md

```diff
@@ -13,224 +13,215 @@ redirect_from:
 
-If you are satisfied with your algorithm and settings, go ahead and hit publish. Congratulations, you’re an algorithm developer!
+If you are satisfied with your algorithm code, behavior, and settings, go ahead and hit "Publish". Congratulations, you’re an algorithm developer!
 
 ### Editing Your Algorithm
 
-Your published algorithm can be edited from the browser, where you can edit the source code, save your work, compile, and submit the algorithm to be available through the API.
-You can also <a href="{{site.baseurl}}/algorithm-development/source-code-management#editing-your-algorithm-source-locally">use Git to push directly to Algorithmia</a> from your current workflow.
+Your published algorithm can be edited from the browser, where you can modify the source code, save your work, compile, and publish the algorithm to be available through the API. You can also <a href="{{site.baseurl}}/algorithm-development/source-code-management#editing-your-algorithm-source-locally">use Git to push directly to Algorithmia</a> from your current workflow.
 
 ### Calling Other Algorithms
 
-Because any call made from your algorithm is considered the same session as when the user calls your algorithm, you won't need to set an API key within your algorithm code. Instead of setting the API key on the client like you would when consuming an algorithm, you can call another from inside your algorithm and pass the input like so:
+When an algorithm is called from within another algorithm, the algorithm called internally is considered to have been made from within the same session as the first algorithm call. Because of this, you won't need to set an API key within your algorithm code. For example, instead of setting the API key on the client like you would when consuming an algorithm, you can call an algorithm internally and pass the input like so:
 
 {% highlight python %}
 if 'url' in input:
   text = Algorithmia.algo('util/Html2Text/0.1.3').pipe(input['url']).result
 {% endhighlight %}
 
-This example shows an algorithm that checks the type of input passed in, and if it is a URL, will call into the Html2Text algorithm. The original caller of your algorithm will be charged for both the first algorithm call as well as the internal algorithm call. The algorithm will automatically use the credentials of the person calling your algorithm.
+This example shows an algorithm that checks the type of input passed in, and if it is a URL, calls the Html2Text algorithm to pull down code from the webpage. Compute associated with both the original algorithm call and the internal algorithm call is tracked and attributed to the original caller's user account.
```

Comment:
```suggestion
This example shows an algorithm that checks the type of input passed in, and if it is a URL, calls the Html2Text algorithm to pull down code from the webpage. Compute associated with both the original algorithm call and the internal algorithm call is tracked and attributed to the original caller's account.
```

Reply:
Just because we don't want to confuse "user account" for organization owned assets

Reply:
I accidentally resolved before committing. So I think this still needs to be committed.